### PR TITLE
Update of definitions for all anatomical planes

### PR DESCRIPTION
**Pull request by UlrikeS91**
@aeidi89 and I noticed that the definitions for the anatomical planes is very human-centered and not universally applicable. I adjusted them a bit to make them more generic and fit all use cases better. 

Examples: 
1. Axial/Horizontal plane had "head to tail" in definition, which is problematic for rodents. There head to tail has a different orientation than a human has (basically the coronal plane of a human instead).
2. Limiting the definitions to "body" is also a bit problematic. Anatomical planes do need a reference but it is not always the whole body. I added "or parts of the body" to make it more open.

### CODE REVIEW

**Review by lzehl — APPROVED** [review]

File: instances/anatomicalPlane/axial.jsonld [hunk 1]
```diff
@@ -4,7 +4,7 @@
   },
   "@id": "https://openminds.ebrains.eu/instances/anatomicalPlane/axial",
   "@type": "https://openminds.ebrains.eu/controlledTerms/AnatomicalPlane",
-  "definition": "An anatomical plane which divides the body into cranial and caudal (head and tail) parts.",
+  "definition": "An anatomical plane which divides the body or parts of the body into superior and inferior (dorsal and ventral, or top and bottom) parts.",
```
Comment (lzehl) [inline instances/anatomicalPlane/axial.jsonld]: Suggestion: The 'axial plane' is always perpendicular to the longitudinal axis of an organism, extremity or organ. Depending on the organisms anatomy this plane divides it's body into superior and inferior, dorsal and ventral, or top and bottom parts.

File: instances/anatomicalPlane/coronal.jsonld [hunk 2]
```diff
@@ -4,7 +4,7 @@
   },
   "@id": "https://openminds.ebrains.eu/instances/anatomicalPlane/coronal",
   "@type": "https://openminds.ebrains.eu/controlledTerms/AnatomicalPlane",
-  "definition": "An anatomical plane which divides the body into divides the body into dorsal and ventral (back and front, or posterior and anterior) parts.",
+  "definition": "An anatomical plane which divides the body or parts of the body into rostral and caudal (anterior and posterior, or front and back) parts.",
```
Comment (lzehl) [inline instances/anatomicalPlane/coronal.jsonld]: Rename to frontal plane? or add this only as synonym?

Suggestion: The 'coronal plane' divides the body and extremities of an organism into dorsal and ventral parts and an organ into anterior and posterior parts.

(I'm not happy with that one)

File: instances/anatomicalPlane/sagittal.jsonld [hunk 3]
```diff
@@ -4,7 +4,7 @@
   },
   "@id": "https://openminds.ebrains.eu/instances/anatomicalPlane/sagittal",
   "@type": "https://openminds.ebrains.eu/controlledTerms/AnatomicalPlane",
-  "definition": "An anatomical plane which divides the body into right and left parts.",
+  "definition": "An anatomical plane which divides the body or parts of the body into left and right (and/or medial and lateral) parts.",
```
Comment (lzehl) [inline instances/anatomicalPlane/sagittal.jsonld]: Suggestion: The 'sagittal plane' refers to any plane that runs parallel to the 'median plane' which divides a symmetrical organism or organ into it's left and right parts.

**Review by lzehl — CHANGES_REQUESTED** [review]
we seem to also miss the third main plane: horizontal / transverse plane. We should add this or even replace the axial plane with it I think. What do you think?

File: instances/anatomicalPlane/sagittal.jsonld (hunk 3 above)
Comment (aeidi89) [reply instances/anatomicalPlane/sagittal.jsonld]: Regarding the 'sagittal plane', be aware of the 'neurological versus radiological' display convention (see link: https://nipy.org/nibabel/neuro_radio_conventions.html). 
In short, "Neurologists like looking at brain images with the patient’s right on the right of the image" versus "Radiologists like looking at their images with the patient’s left on the right of the image".

At the moment there are no issue as you've stated "...organism or organ into it's left and right parts", and thus following the neurological display convention, however it is useful to be aware.

**Comment by lzehl** [comment]
I'm gonna take this one over together with @aeidi89 ; thanks @UlrikeS91 for triggering this (this is a tricky one)
@tgbugs any ideas?

File: instances/anatomicalPlane/axial.jsonld (hunk 1 above)
Comment (aeidi89) [reply instances/anatomicalPlane/axial.jsonld]: Suggestion:
The 'axial plane' is always perpendicular to the longitudinal and the frontal axes of an organism, extremity or organ, dividing it into a dorsal, superior, above or top part, and a ventral, inferior, below or bottom part.

Link to webpage: https://www.healthline.com/health/body-planes#transverse-plane
 We could use this as a reference too, but I don't know: https://www.frontiersin.org/articles/10.3389/fninf.2023.1154080/full

File: instances/anatomicalPlane/axial.jsonld [hunk 4]
```diff
@@ -4,7 +4,7 @@
...
   "description": null,
   "interlexIdentifier": null,
   "knowledgeSpaceLink": null,
```
Comment (aeidi89) [inline instances/anatomicalPlane/axial.jsonld:10]: Could we add synonym?

  "synonym": [
    "Transverse",
    "Horizontal"
  ]

File: instances/anatomicalPlane/coronal.jsonld (hunk 2 above)
Comment (aeidi89) [reply instances/anatomicalPlane/coronal.jsonld]: I agree, this should be renamed to frontal.

Suggestion:
The 'frontal plane' is always perpendicular to the axial and longitudinal axes of an organism, extremity or organ, dividing it into an anterior, rostral or front part, and a posterior, caudal or back part.

Link to webpage: https://www.healthline.com/health/body-planes#coronal-plane
We could use this as a reference too, but I don't know: https://www.frontiersin.org/articles/10.3389/fninf.2023.1154080/full


Could we add synonym?

"synonym": [
"Coronal"
]

File: instances/anatomicalPlane/sagittal.jsonld (hunk 3 above)
Comment (aeidi89) [reply instances/anatomicalPlane/sagittal.jsonld]: Too keep up with the two other axes, should this be renamed to Longitudinal?

Suggestion:
The 'longitudinal plane' is always perpendicular to the axial and frontal axes of an organism, extremity or organ, running parallel to the 'median plane' and dividing it into a left and right part. 

Link to webpage: https://www.healthline.com/health/body-planes#sagittal-plane
We could use this as a reference too, but I don't know: https://www.frontiersin.org/articles/10.3389/fninf.2023.1154080/full

Could we add synonym?

"synonym": [
"Sagittal"
]

**Review by aeidi89 — CHANGES_REQUESTED** [review]
I have made comments with suggestion in all three instances.

@lyuba, if you agree, please make the changes and merge.
If not, we should meet and finalize it together.

**Comment by lzehl** [comment]
@aeidi89 let's go over this one tomorrow together. It's already a while ago that I digged into the topic...

**Review by lzehl — APPROVED** [review]

**Review by aeidi89 — APPROVED** [review]
Looks good

**Pull request merged**